src/meta/processors/admin/CreateBackupProcessor.cpp

```diff
@@ -178,12 +186,11 @@ void CreateBackupProcessor::process(const cpp2::CreateBackupReq& req) {
     return;
   }
 
-  // step 6 : update snapshot status from INVALID to VALID.
+  // step 6 : update backup status from INVALID to VALID.
   data.emplace_back(
```

Comment:
data has been moved in line 135, reuse it might trigger some bug

Reply:
you are right